src/router/graphql.ts

```diff
@@ -36,6 +37,12 @@ if (config.graphql) {
             externalUrlBase: `/v1/graphql/${db}`,
             graphiql: withGraphiql,
             watchPg: true,
+            handleErrors: (errors, _req, _res) => {
+              for (const error of errors) {
+                logErrSentry(error, uid, email, db);
+              }
+              return errors;
+            }
```

Comment:
Re https://github.com/iasql/iasql-engine/pull/959#issuecomment-1180321884, this would have to be out of the loop for that too happen I think?

Reply:
hmm errors is just an array of one element here :/